cmd/release-controller/http_helper.go

```diff
@@ -873,3 +881,19 @@ func checkReleasePage(page *ReleasePage) {
 		}
 	}
 }
+
+func (c *Controller) getDashboards() []Dashboard {
```

Comment:
If you don't need to call this in multiple places, just inline it in intialization.  Taking references to controller methods in a function is wierd because different callers might get different results down the road.

Reply:
I have inlined the initialization of the slice. and removed the passing of functions as an argument.